public/index.html

```diff
@@ -1,37 +1,176 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-  </body>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<meta name="keywords" content="music, spotify, DONT BlOCK SPOTIFY IN RUSSIA" />
+<meta name="description" content="Spotify redesign" />
+<link href="../src/index.css" rel="stylesheet" type="text/css" />
+<link href="../src/img/Spotify.ico" rel="shortcut icon" type="image/x-icon" />
+<title>Spotify - веб приложение</title>
+</head>
+<body>
+
+	<header>
+    <a id="MainPage" href = "main.html"> Main page</a>
+    <img  class="logo" src="../src/img/spotify-logo.svg" title="Spotify logo" alt="Spotify logo" height="90%" width="90%"/>
```

Comment:
Не корректно использовать подобный пути.
стоит поместить картинки рядом с index.html, и сделать что-то такое
```suggestion
    <img  class="logo" src="%PUBLIC_URL%/img/spotify-logo.svg" title="Spotify logo" alt="Spotify logo" height="90%" width="90%"/>
```